sashimi/state.py

```diff
@@ -244,13 +249,16 @@ def get_voxel_size(
 
 def convert_save_params(
     save_settings: SaveSettings,
-    scanning_settings: ZRecordingSettings,
+    scanning_settings: Union[ZRecordingSettings, SinglePlaneSettings],
     camera_settings: CameraSettings,
     trigger_settings: TriggerSettings,
 ):
-    n_planes = scanning_settings.n_planes - (
-        scanning_settings.n_skip_start + scanning_settings.n_skip_end
-    )
+    if isinstance(scanning_settings, SinglePlaneSettings):
+        n_planes = 0
+    else:
+        n_planes = scanning_settings.n_planes - (
+            scanning_settings.n_skip_start + scanning_settings.n_skip_end
+        )
```

Comment:
Inside `convert_save_params` checks if the settings are for planar mode, and if so,  sets n_planes to 0